curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/65822bd82d708c4895080c35.md

```diff
@@ -7,14 +7,14 @@ dashedName: step-9
 
 # --description--
 
-You can access values in a dictionary through its keys. You need to use the bracket notation and include the key between the square brackets:
+You can access values in a dictionary through its keys. You need to use bracket notation and include the key between the square brackets:
 
 ```py
 my_dict = {'amount': 50.0, 'category': 'Food'}
 my_dict['amount'] # 50.0
 ```
 
-You are currently interpolating the `expense` dictionary in your f-string. Modify the f-string expression to access the value of the `'amount'` key and the `'category'` key in the `expense` dictionary.
+You are currently interpolating the `expense` dictionary in your f-string. Modify the f-string expression to access the value of the `'amount'` key and the `'category'` key in the `expense` dictionary. Remember to use double quotes within your single-quoted f-string and vice versa.
```

Comment:
We should ensure that quotation rules are explained in one of the project before.
Discussion [here](https://github.com/freeCodeCamp/freeCodeCamp/issues/53542)

Reply:
Suggestion: maybe we can introduce the quotation rules here in this step.